TopMovies/API/GraphQL/NetworkManager.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  NetworkManager.swift
+//  TopMovies
+//
+//  Created by Macbook Pro  on 01.04.2021.
+//
+
+import Foundation
+
+struct QueryJSON: Encodable {
+  let query: String
+}
+
+class NetworkManager {
+  func requestAsync(with queryJSON: QueryJSON, completion: @escaping (Result<Data, Error>) -> Void) {
+    let session = URLSession(configuration: .default)
+    var request = URLRequest(url: URL(string: "https://tmdb.apps.quintero.io/")!)
+    request.httpMethod = "POST"
+    request.setValue("application/json", forHTTPHeaderField: "Content-Type")
+    
+    do {
+      let jsonData = try JSONEncoder().encode(queryJSON)
+      request.httpBody = jsonData
+    } catch {
+      completion(.failure(error))
+    }
+    
+    DispatchQueue.global(qos: .userInitiated).async {
```

Comment:
I'd prefer to avoid using `.global()` queues since you never know who can use that queue and block your code using some barriers and so on.
Which is why:

```swift
let queue = DispatchQueue(label: "com.topMovies.network"
```